tests/lookup/tests.py

```diff
@@ -189,6 +189,18 @@ def test_in_bulk_with_field(self):
             }
         )
 
+    def test_in_bulk_with_constraint(self):
```

Comment:
It might be worthwhile to include a test for a `UniqueConstraint` spanning two fields.